dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/Provisioner.java

```diff
@@ -399,6 +401,9 @@ private Bundle installFeatureBundle(String urlString, String productName, Bundle
                     Region productRegion = getProductRegion(productName);
                     // Bundle will just be returned if something from this location exists already.
                     bundle = productRegion.installBundleAtLocation(location, new URL(urlString).openStream());
+                    if (ActivationType.parallel.equals(fr.getActivationType())) {
```

Comment:
ActivationType.parallel would appear from here to be a static final field, so why isn't it upper case as per the normal Java convention?

Reply:
Changed to uppercase constants